lib/rules/no-element-handle.js

```diff
@@ -0,0 +1,77 @@
+function isPageIdentifier({ callee }) {
+  return (
+    callee &&
+    callee.type === 'MemberExpression' &&
+    callee.object.type === 'Identifier' &&
+    callee.object.name === 'page'
+  );
+}
+
+function isElementHandleIdentifier({ callee }) {
+  return (
+    callee &&
+    callee.property &&
+    callee.property.type === 'Identifier' &&
+    callee.property.name === '$'
+  );
+}
+
+function isElementHandlesIdentifier({ callee }) {
+  return (
+    callee &&
+    callee.property &&
+    callee.property.type === 'Identifier' &&
+    callee.property.name === '$$'
+  );
+}
+
+function getRange(node) {
+  const awaitLength = 'await '.length;
+  const elementHandleLength = 'page.$'.length;
+  const elementHandlesLength = 'page.$$'.length;
+
+  let start = 0;
+  let end = 0;
+
+  if (node.parent && node.parent.type === 'AwaitExpression') {
+    start = start + node.parent.range[0];
+    end = start + awaitLength;
+  } else {
+    start = start + node.range[0];
+    end += start;
+  }
+
+  if (isElementHandleIdentifier(node)) end += elementHandleLength;
+  if (isElementHandlesIdentifier(node)) end += elementHandlesLength;
+
+  return [start, end];
+}
+
+module.exports = {
+  create(context) {
+    return {
+      CallExpression(node) {
+        if (isPageIdentifier(node) && (isElementHandleIdentifier(node) || isElementHandlesIdentifier(node))) {
+          context.report({
+            messageId: 'noElementHandle',
+            fix: (fixer) => fixer.replaceTextRange(getRange(node), 'page.locator'),
```

Comment:
Because auto fixing this would result in incorrect code in the case of `$$`, we probably should use the [ESLint suggestions API](https://eslint.org/docs/developer-guide/working-with-rules#providing-suggestions) instead.

Reply:
Sure, I'll change it to the suggestion.